Argo/Types/JSON.swift

```diff
@@ -13,14 +13,7 @@ public extension JSON {
   static func parse(json: AnyObject) -> JSON {
     switch json {
     case let v as [AnyObject]: return .Array(v.map(parse))
-
-    case let v as [Swift.String: AnyObject]:
-      return .Object(reduce(v, [:]) { (var accum, elem) in
-        let parsedValue = (self.parse <^> elem.1) ?? .Null
-        accum[elem.0] = parsedValue
-        return accum
-      })
-
+    case let v as [Swift.String: AnyObject]: return .Object(parse <^> v)
```

Comment:
I'd like to suggest that the line above should also use the map operator.

Otherwise, I can remove my use of it for consistency which would also mean that this file no longer depends on Runes.